public/video-ui/src/components/utils/VideoEmbed.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+
+export function VideoEmbed({ sources }) {
+  return (
+    <video className="video-player" controls>
+      {sources.map(source => {
+        return (
+          <source key={source.src} src={source.src} type={source.mimeType} />
```

Comment:
We can set the `src` of the `video` tag. I believe this wins over the `source` set, so would have to be either or rather than setting both.

Something like:

```js
if (sources.length === 1) {
  <video src={source.src}>
} else {
  <video>
    {sources.map(source => <source src={source.src}>}
  </video>
}
```

should do the trick.